tests/compiler/cbs/accumulator_for.cpp

```diff
@@ -2,6 +2,10 @@
 // RUN: %t | FileCheck %s
 // RUN: %acpp %s -o %t --acpp-targets=omp --acpp-use-accelerated-cpu -O
 // RUN: %t | FileCheck %s
+// RUN: %acpp %s -o %t --acpp-targets=generic --acpp-use-accelerated-cpu
+// RUN: %t | FileCheck %s
+// RUN: %acpp %s -o %t --acpp-targets=generic --acpp-use-accelerated-cpu -O
```

Comment:
Are you sure this actually runs on host backend? When using `generic` you need to be careful, since it can pick whatever device it finds. It also prioritizes OpenCL CPU backend over OpenMP backend typically.
I think in CI it's probably fine since in the CBS lit workflow we don't install other backends. But it might not do the expected thing when run locally.
So better use `ACPP_VISIBILITY_MASK=omp` or explicitly select the OpenMP device programmatically.